config/initializers/assets.rb

```diff
@@ -1,3 +1,5 @@
-if Rails.application.config.should_show_easy_login
-  Rails.application.config.assets.precompile += %w( admin.css public.css easy_sign_in.js )
-end
+Rails.application.config.assets.precompile += %w(
+  admin.css
+  easy_sign_in.js
```

Comment:
@pzi give me one good reason not to precompile this tiny damn file for all ;)

It'll sit on the server being available and if required, be loaded.  If not required, won't be loaded and no harm is done. 

Conditionals in the precompile list based on env are problematic.

If you consider that the correct way to do asset precompilation is execute it on your local machine and push the result at the server, an ENV dependent precompile list will cause issues